site/app/templates/forum/showFullThreadsPage.twig

```diff
@@ -46,7 +46,9 @@
         <div id="thread_list" data-prev_page="{{ prev_page }}" data-next_page="{{ next_page }}">
             <i class="fas fa-spinner fa-spin fa-2x fa-fw fill-available" style="color:gray;display: none;" aria-hidden="true"></i>
             <i class="fas fa-caret-up fa-2x fa-fw fill-available" style="color:gray; display: none" aria-hidden="true"></i>
-
+            <script>
```

Comment:
Do you need to set a cookie for this? Seems like it would be possible to have twig create a global js variable since the line below is defining `is_full_page` to `true` as a twig variable.

Something like

```
<!---- presence of this data var indicates full page --->
<div id="variable-storage" data-my-variable="is-full-page" style="display: none;"></div>
```

or

```
<script>
    const isFullPage = true;
</script>
```

to also be outputed by a twig template to get the information you need to the frontend